src/balances/balances.controller.spec.ts

```diff
@@ -125,19 +126,19 @@ describe('Balances Controller (Unit)', () => {
             return Promise.resolve({
               data: transactionServiceBalancesResponse,
             });
-          } else if (url == 'https://test.exchange') {
-            return Promise.reject({ status: 500 });
+          } else if (url == 'https://test.exchange/latest') {
+            return Promise.reject({ status: HttpStatus.INTERNAL_SERVER_ERROR });
           } else {
             return Promise.reject(new Error(`Could not match ${url}`));
           }
         });
 
         await request(app.getHttpServer())
           .get(`/chains/${chainId}/safes/${safeAddress}/balances/usd`)
-          .expect(503)
+          .expect(HttpStatus.SERVICE_UNAVAILABLE)
```

Comment:
I get your point. These codes are HTTP statuses which won't never change, and most likely we won't be using our our codes instead of the protocol ones, but I get your point about being more explicit in the tests and remove dependencies. Keeping the numeric value in the test, and the enum reference in the production code, it's also a safe net for human errors setting the codes, so I will keep numeric codes in the tests 👍